coredb-operator/src/controller.rs

```diff
@@ -133,45 +133,47 @@ impl CoreDB {
                 if !is_pod_ready().matches_object(Some(&primary_pod)) {
                     debug!("Did not find primary pod");
                     return Ok(Action::requeue(Duration::from_secs(1)));
+                } else {
+                    // coredb is ready
+                    let patch_status = json!({
+                        "status": {"running": true}
+                    });
+                    patch_cdb_status_merge(&coredbs, &name, patch_status).await?;
                 }
```

Comment:
I feel like we should always set `status.running` here too. cc @shahadarsh , maybe we dont need it?